pkg/repo/mysql/message.go

```diff
@@ -0,0 +1,150 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func (r *Repo) SaveMessages(ctx context.Context, serviceID uuid.UUID, messages *model.Messages) error {
+	_, err := r.LoadService(ctx, serviceID)
+	if err != nil {
+		return fmt.Errorf("repo: load service: %w", err)
+	}
+
+	tx, err := r.db.BeginTx(ctx, nil)
+	if err != nil {
+		return fmt.Errorf("repo: begin tx: %w", err)
+	}
+
+	defer tx.Rollback() //nolint:errcheck
+
+	// Check if message already exist
+	var messageID uuid.UUID
+
+	row := tx.QueryRowContext(
+		ctx,
+		`SELECT id FROM message WHERE service_id = UUID_TO_BIN(?) AND language = ?`,
+		serviceID.String(),
```

Comment:
This should work. `uuid.UUID` implements the `sql.Valuer` interface. I don't know about `language.Tag`.

```suggestion
		serviceID,
```

Reply:
yes, you are right, uuid has both Value() and Scan() methods.
Language tag does not